app/src/main/java/com/github/libretube/ui/dialogs/ColorPickerDialog.kt

```diff
@@ -0,0 +1,133 @@
+package com.github.libretube.ui.dialogs
+
+import android.app.Dialog
+import android.content.Context
+import android.graphics.Color
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import android.widget.SeekBar
+import android.widget.Toast
+import androidx.fragment.app.DialogFragment
+import com.github.libretube.R
+import com.github.libretube.databinding.DialogColorPickerBinding
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+
+class ColorPickerDialog(
+    private val context: Context,
+    private val initialColor: Int,
+    private val onColorSelectedListener: OnColorSelectedListener
+) : DialogFragment(), SeekBar.OnSeekBarChangeListener, View.OnClickListener {
+
+    private lateinit var binding: DialogColorPickerBinding
```

Comment:
The view binding has to be freed in the `onDestroyView` method: https://developer.android.com/topic/libraries/view-binding#fragments

```suggestion
    private var _binding: DialogColorPickerBinding? = null
    private val binding get() = _binding!!
```

Reply:
Why do we have two binding vars here(still new to kotlin)?